src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java

```diff
@@ -500,6 +502,22 @@ public ColorRGB[] executeSimple(Object[][] params) {
 		}).description("Returns a RGB color from the given red, green and blue parameters.")
 			.examples("dye player's leggings rgb(120, 30, 45)")
 			.since("2.5");
+
+		Functions.registerFunction(new SimpleJavaFunction<OfflinePlayer>("player", new Parameter[] {
+			new Parameter<>("nameOrUUID", DefaultClasses.STRING, true, null)
+		}, DefaultClasses.OFFLINE_PLAYER, true) {
+			@Override
+			public OfflinePlayer[] executeSimple(Object[][] params) {
+				String name = (String) params[0][0];
+				UUID uuid = null;
+				try {
+					uuid = UUID.fromString(name);
```

Comment:
It would be much better to perform (at least some) of the basic is-it-a-UUID checks yourself rather than asking for an exception. E.g. is it too long to be a username? Does it contain the `-` character?

Reply:
I think that's redundant, the usage of UUID.fromString will remain as is and exception handling will remain right?

Reply:
yeah it's all just a string object no uuid classinfo